packages/in_app_purchase/in_app_purchase_storekit/lib/src/in_app_purchase_storekit_platform.dart

```diff
@@ -251,11 +251,17 @@ class InAppPurchaseStoreKitPlatform extends InAppPurchasePlatform {
   @Deprecated('Use countryCode')
   Future<String?> getCountryCode() => countryCode();
 
-  /// Turns on StoreKit2. You cannot disable this after it is enabled.
-  /// This can only be enabled if your device supports StoreKit 2.
+  /// StoreKit 2 is now the default.
+  @Deprecated('StoreKit 2 is now the default')
   static Future<bool> enableStoreKit2() async {
-    _useStoreKit2 = await SKRequestMaker.supportsStoreKit2();
-    return _useStoreKit2;
+    _useStoreKit2 = true;
+    return true;
+  }
+
+  /// Call this before `registerPlatform` to re-enable StoreKit1
+  static bool enableStoreKit1() {
```

Comment:
should we also deprecate this with a message that storekit 1 will be removed in the future (if that's the plan).